src/browser/api_protocol/index.ts

```diff
@@ -24,8 +24,13 @@ const InterApplicationBusApiHandler = require('./api_handlers/interappbus').Inte
 const NotificationApiHandler = require('./api_handlers/notifications').NotificationApiHandler;
 const SystemApiHandler = require('./api_handlers/system').SystemApiHandler;
 const WindowApiHandler = require('./api_handlers/window').WindowApiHandler;
+import { init as initApiProtocol, getDefaultRequestHandler } from './api_handlers/api_protocol_base';
```

Comment:
:neckbeard: I think it may make sense to avoid `as` renaming (makes harder to refactor if one side is JS), though the more things that are TS the less it matters... so basically dont change a thing :)